packages/api-client/src/components/Sidebar/Sidebar.vue

```diff
@@ -158,7 +154,5 @@ watch(
   width: 100%;
   height: 100%;
   z-index: 100;
-  backdrop-filter: blur(1.5px);
-  transition: backdrop-filter 0.15s ease-in-out;
```

Comment:
before element is to be removed if we don't rely on backdrop filter / transition